### PR TITLE
Bsp/fastpass integration test

### PR DESCRIPTION
**Pull request by lukaszwawrzyk**

### CODE REVIEW

File: extensions/scala/tests/src/test/scala/org/virtuslab/intellij/extensions/PantsBspImportTest.scala [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+        "java_app__scala_tests-root"
+      )
+      Assert.assertEquals("Module set is incorrect", expectedModules, project.modules.map(_.name).toSet)
+    }
+  }
+
+
+  private def createBspProjectWithFastpass(workspace: Path, targets: String*): Path = {
+    val args = Seq(
+      "create",
```
Comment (tpasternak) [inline extensions/scala/tests/src/test/scala/org/virtuslab/intellij/extensions/PantsBspImportTest.scala:36]: We could add `--name` parameter here

File: driver/sources/src/main/scala/org/virtuslab/ideprobe/RobotExtensions.scala [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+import com.intellij.remoterobot.fixtures.Fixture
+import com.intellij.remoterobot.search.locators.Locators
+import Extensions._
+
+trait RobotExtensions {
+  protected val robotTimeout = Duration.ofSeconds(10)
+
+  implicit class SearchableOps(val component: SearchContext) {
+    def find(xpath: String): CommonContainerFixture = {
+      component.find(classOf[CommonContainerFixture], Locators.byXpath(xpath), robotTimeout)
```
[the remark this answers is not in the document]
Comment (lukaszwawrzyk) [reply driver/sources/src/main/scala/org/virtuslab/ideprobe/RobotExtensions.scala:16]: Exception with timeout on which element it failed

File: driver/sources/src/main/scala/org/virtuslab/ideprobe/RobotExtensions.scala [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+        case many => throw new RuntimeException(s"Found multiple elements for query $xpath: $many")
+      }
+    }
+
+    def mainWindow: CommonContainerFixture = find(query.className("IdeFrameImpl"))
+  }
+
+  implicit class FixtureOps(val component: Fixture) {
+    def fullText: String = fullTexts.mkString("\n")
+    def fullTexts: Seq[String] = component.findAllText.asScala.map(_.getText).toList
```
[the remark this answers is not in the document]
Comment (lukaszwawrzyk) [reply driver/sources/src/main/scala/org/virtuslab/ideprobe/RobotExtensions.scala:36]: Fixture is just a base class with this method, generally it is actually a component. This is visible text on the component

File: extensions/scala/tests/src/test/scala/org/virtuslab/intellij/extensions/ScalaImportTest.scala [hunk 4]
```diff
@@ -4,34 +4,33 @@ import org.junit.Assert
...
-  DependencyProvider.registerBuilder(ScalaPluginBuilder)
+final class ScalaImportTest extends ScalaPluginTestSuite {
 
   private val config = Config.fromClasspath("SbtProject/ideprobe.conf")
 
   @Test
   def importSbtProject(): Unit = {
-    IntelliJFixture.fromConfig(config).run { intellij =>
+    fixtureFromConfig(config).run { intellij =>
+      intellij.probe.preconfigureJDK()
```
[the remark this answers is not in the document]
Comment (lukaszwawrzyk) [reply extensions/scala/tests/src/test/scala/org/virtuslab/intellij/extensions/ScalaImportTest.scala:19]: I just wanted to have green tests, I worried if we can publish probe with broken tests (can we?). I want to remove it as soon as Justin merges your PR.

**Pull request merged**